hathor/cli/run_node.py

```diff
@@ -201,7 +206,11 @@ def prepare(self, *, register_resources: bool = True) -> None:
 
         if self._args.stratum:
             assert self.manager.stratum_factory is not None
-            self.reactor.listenTCP(self._args.stratum, self.manager.stratum_factory)
+
+            if self._args.x_enable_ipv6:
+                self.reactor.listenTCP(self._args.stratum, self.manager.stratum_factory, interface='::0')
+            else:
+                self.reactor.listenTCP(self._args.stratum, self.manager.stratum_factory)
```

Comment:
Shouldn't we listen in both interfaces (ipv4 and ipv6) when ipv6 is enabled?

Reply:
Initially I had this impression too. But it turns out Linux by default will map IPv4 to IPv6, and listening on `::0` is enough to handle IPv4 traffic as well.

I was actually getting an error when trying to listen on IPv4 simultaneously on the same port.

You can reproduce this behavior by running a simple server like this listening on `::`:

```
python3 -m http.server --bind :: 8080
```

And then connecting to it both over IPv6 or IPv4:

```
nc 127.0.0.1 8080
nc ::1 8080
```

There is a kernel paramenter to control this behavior, but the default is to translate IPv4 to IPv6 when bound to `::0`: https://sysctl-explorer.net/net/ipv6/bindv6only/

Reply:
Done

Reply:
Thanks!